lenra-io/server
# fix: OAuth2 client custom scheme redirect URI

For Android and iOS clients





## About this PR
We can't create OAuth2 client with a custom scheme in the redirect URI (ex: `io.lenra.app:/`), but it's needed for Android ans iOS applications.

This PR fixes it.

## Technical highlight/advice
Just changed the corresponding Regex.

## How to test my changes

- start the Docker Compose elements: `docker-compose up -d`
- create the backoffice OAuth2 client (see `docs/oauth-testing.md`): `mix create_oauth2_client backoffice`
- start the backoffice
- create an app
- create a OAuth2 client for this app in the backoffice with this value in redirect URI: `io.lenra.app:/`


## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
